dot/import.go

```diff
@@ -127,7 +127,11 @@ func newHeaderFromFile(filename string) (*types.Header, error) {
 			return nil, err
 		}
 
-		err = digest.Add(digestItem.Value())
+		digestItemVal, err := digestItem.Value()
+		if err != nil {
+			return nil, err
```

Comment:
nit (also adding the log index would be nice)

```suggestion
			return nil, fmt.Errorf("getting digest item value: %w", err)
```